### PR TITLE
LPS-125550 0224 run test

### PR DESCRIPTION
**Pull request by yuhai**

### CODE REVIEW

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

File: portal-web/build.xml [hunk 1]
```diff
@@ -106,6 +106,7 @@
 			outputproperty="jspc.output"
 		>
 			<jvmarg if:true="${jvm.debug}" line="${jpda.settings}" />
+			<sysproperty key="log4j2.disable.jmx" value="true" />
```
Comment (tinatian) [inline portal-web/build.xml:109]: @yuhai @dantewang Please add this property to system.properties, it should also work.

Comment (yuhai) [reply portal-web/build.xml:109]: @tinatian, got it. worked.

**Pull request closed without merging**